Assets/Scripts/settingUI.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class settingUI : MonoBehaviour
+
+{
+    string pname;
+    public Light sunLight;
+    GameObject sliderInitialValue;
+    void start()
+    {
+        sunLight = GetComponent<Light>();
+      
+
+    }
+   public void setBrightness(float sliderValue)
+    {
+        
+        sunLight.intensity = sliderValue * 2;
```

Comment:
Brightness should be changed through *post-processing* effects, not the scene light directly. Post-processing changes the camera's perception of the scene, which means everything will be affected uniformly.